diff --git a/src/Board.js b/src/Board.js
index 9f111ff..1f00045 100644
--- a/src/Board.js
+++ b/src/Board.js
@@ -21,30 +21,6 @@ export default class Board extends React.Component {
       complete: React.createRef(),
     }
   }
-
-  componentDidMount(){
-    this.dragula = Dragula([
-      this.swimlanes.backlog.current,
-      this.swimlanes.inProgress.current,
-      this.swimlanes.complete.current
-    ])
-
-    this.dragula.on('drop', (element, target, source, sibling) => {
-      if (target.id === 'Backlog') {
-        element.className = 'Card Card-grey';
-        element.setAttribute("data-status", "Backlog");
-      }
-      else if (target.id === 'In Progress') {
-        element.className = 'Card Card-blue';
-        element.setAttribute("data-status", "in Progress");
-      }
-      else if (target.id === 'Complete') {
-        element.className = 'Card Card-green';
-        element.setAttribute("data-status", "Complete");
-      }
-    })
-  }
-
   getClients() {
     return [
       ['1','Stark, White and Abbott','Cloned Optimal Architecture', 'in-progress'],
@@ -74,7 +50,6 @@ export default class Board extends React.Component {
       status: companyDetails[3],
     }));
   }
-
   renderSwimlane(name, clients, ref) {
     return (
       <Swimlane name={name} clients={clients} dragulaRef={ref}/>
diff --git a/src/Swimlane.js b/src/Swimlane.js
index f58b476..7e6e0d0 100644
--- a/src/Swimlane.js
+++ b/src/Swimlane.js
@@ -18,7 +18,7 @@ export default class Swimlane extends React.Component {
     return (
       <div className="Swimlane-column">
         <div className="Swimlane-title">{this.props.name}</div>
-        <div className="Swimlane-dragColumn" id={this.props.name} ref={this.props.dragulaRef}>
+        <div className="Swimlane-dragColumn" ref={this.props.dragulaRef}>
           {cards}
         </div>
       </div>);
